client/v2/autocli/query_test.go

```diff
@@ -121,6 +130,20 @@ func TestCoin(t *testing.T) {
 	assert.DeepEqual(t, conn.lastRequest, conn.lastResponse.(*testpb.EchoResponse).Request, protocmp.Transform())
 }
 
+func TestMap(t *testing.T) {
+	conn := testExecCommon(t, buildModuleQueryCommand,
+		"echo",
+		"1",
+		"abc",
+		"1234foo",
+		"4321bar",
+		"--map-string-string", "foo=bar",
```

Comment:
Can we test edge cases, like a string containing a comma? It should fail given the current implementation f.e.